### DISCUSSION TITLE
Passing Through SMB Mount to use as storage for all recordings

### DISCUSSION THREAD

**Discussion by LampsPost** [blakeblackshear/frigate#15071] [Config Support]
I have managed to create a mount point within Proxmox LXC for my synology SMB share, I can't workout how to change the volume frigate is using to the SMB mounted volume. I have seen several routes using docker but I can't work out how to do this using the Proxmox shell, I'm confident I'm using the command or missing something basic

Thanks in advance for your help

**Comment by moogoos** [comment]
I would recommend going to the Proxmox forum or reddit for help. Perhaps: https://forum.proxmox.com/threads/tutorial-unprivileged-lxcs-mount-cifs-shares.101795/

The 3 paths that are important are:
```
#      device: ":...../recordings"
#      device: ":...../clips"
#      device: ":...../exports"

as 

/media/frigate/recordings
/media/frigate/clips
/media/frigate/exports

```

**Reply by LampsPost** [reply]
Thank you, I will give that a try

**Reply by moogoos** [reply]
Here is another resource: https://www.youtube.com/watch?v=_cCeKd-49WM. 

I have been anti LXC in Proxmox because you're messing with the hypervisor directly in A LOT of cases, but to each their own. If you move to a VM with containers I can help you as well. My primary config is using bind mounts in the container in docker in the container definittion directly. If you manage shares at the host level you're going to have lots of issues because the container doesn't typically know when it isn't available. It's just perceived as a path from the container on the host.
